### DISCUSSION TITLE
Cellular Growth | Solidification Modeling

### DISCUSSION THREAD

**Discussion by rl3fz** [idaholab/moose#18717] [Q&A Modules: Phase field] [answered]
Hello Everyone!

I'm looking to simulate cellular growth during solidification. Does MOOSE have any kernels that would be particularly helpful here? I'm familiar with the grand potential examples, but I'm looking for more control over the direction/orientation (and if possible, type) of the growth.

Any and all advice would be greatly appreciated!

**Comment by GiudGiud** [comment]
@laagesen

**Comment by SudiptaBiswas** [comment, marked as answer]
Grand potential-based solidification model is what we have that can be used here. You will be able to control orientation and growth type by selecting appropriate interfacial properties.

**Reply by rl3fz** [reply]
Got it, thank you!
